app/Filament/Pages/Installer/PanelInstaller.php

```diff
@@ -32,54 +34,43 @@ class PanelInstaller extends SimplePage implements HasForms
 {
     use CheckMigrationsTrait;
     use EnvironmentWriterTrait;
-    use HasUnsavedDataChangesAlert;
     use InteractsWithForms;
 
     public $data = [];
 
     protected static string $view = 'filament.pages.installer';
 
+    private User $user;
+
     public function getMaxWidth(): MaxWidth|string
     {
         return MaxWidth::SevenExtraLarge;
     }
 
-    public static function show(): bool
+    public static function isInstalled(): bool
     {
-        if (User::count() <= 0) {
-            return true;
-        }
```

Comment:
I definitely need this in there, it's a much better user experience.

Reply:
I disagree. This ONLY applies to docker... You're also forcing docker users to use SQLite in the beginning.